needy/__main__.py

```diff
@@ -13,8 +13,21 @@ def satisfy(args=[]):
         description='Satisfies library and universal binary needs.',
         formatter_class=argparse.RawDescriptionHelpFormatter
     )
-    parser.add_argument('--target', default='host', help='builds needs for this target (example: ios:armv7)')
-    parser.add_argument('--universal-binary', help='builds the universal binary with the given name')
+    parser.add_argument(
+        '--target',
+        default='host',
+        help='builds needs for this target (example: ios:armv7)')
+    parser.add_argument(
+        '--universal-binary',
+        help='builds the universal binary with the given name')
+    parser.add_argument(
+        '--jobs',
```

Comment:
Weird. I'm 95% sure I did rename the destination variable.. I'll fix that in a second.